packages/rule-typescript-config-is-valid/src/rule.ts

```diff
@@ -0,0 +1,135 @@
+        };
+
+        /**
+         * Returns a readable error for 'enum' errors.
+         */
+        const generateEnumError = (error: ajv.ErrorObject): string => {
+            const property = error.dataPath.substr(1);
+            const allowedValues = (error.params as ajv.EnumParams).allowedValues;
+
+            return `'${property}' ${error.message} '${allowedValues.join(', ')}'. Value found '${error.data}'`;
+        };
+
+        /**
+         * Returns a readable error for 'pattern' errors.
+         */
+        const generatePatternError = (error: ajv.ErrorObject) => {
+            const property = error.dataPath.substr(1);
+
+            return `'${property}' ${error.message.replace(/"/g, '\'')}. Value found '${error.data}'`;
+        };
+
+        /**
+         * Returns a readable error message.
+         */
+        const prettyfy = (error: ajv.ErrorObject): string => {
+            let result: string;
+
+            switch (error.keyword) {
+                case 'additionalProperties':
+                    result = generateAdditionalPropertiesError(error);
+                    break;
+                case 'enum':
+                    result = generateEnumError(error);
+                    break;
+                case 'pattern':
+                    result = generatePatternError(error);
+                    break;
+                default:
+                    result = error.message;
+                    break;
+            }
+
+            return result;
+        };
+
+        /**
+         * Report all the validation errors received.
+         */
+        const report = async (errors: Array<ajv.ErrorObject>, resource: string) => {
+            for (const error of errors) {
+                /*
+                 * When some of the error is 'anyOf' we need to build the message
+                 * with the other errors.
+                 */
+                if (error.keyword === 'anyOf') {
+                    const otherErrors = _.without(errors, error);
+
+                    const results = otherErrors.map((otherError) => {
+                        return prettyfy(otherError);
+                    });
+
+                    await context.report(resource, null, results.join(' or '));
+                } else {
+                    await context.report(resource, null, prettyfy(error));
+                }
+            }
+        };
+
+        const invalidJSONFile = async (typeScriptConfigInvalid: TypeScriptConfigInvalid) => {
+            const { error, resource } = typeScriptConfigInvalid;
+
+            debug(`Validating rule typescript-config-is-valid`);
+
+            await context.report(resource, null, error.message);
+        };
+
+        const invalidSchema = async (fetchEnd: TypeScriptConfigInvalidSchema) => {
+            const { errors, resource } = fetchEnd;
+
+            debug(`Validating rule typescript-config-is-valid`);
```

Comment:
Maybe something like:

> schema is invalid

`debug` should already prefix where the message is coming from